### PR TITLE
Docker readme server

### PR DESCRIPTION
**Pull request by axfelix**

### CODE REVIEW

**Comment by axfelix** [comment]
the build appears to be failing but I think I just branched main at a bad time...

File: docker-readme/server.md [hunk 1]
```diff
@@ -326,37 +342,63 @@ This variable allows you to pass a frontend security certificate key as an objec
...
 
+This variable allows you to specify an endpoint to connect to [statsd](https://github.com/statsd/statsd) for observability.
+
 `PROMETHEUS_ENDPOINT` (defaults to unset)
 
+This variable allows you to specify an endpoint to connect to Prometheus for observability.
+
 `PROMETHEUS_TIMER_TYPE` (defaults to `histogram`)
 
+This variable specifies which [type of metrics](https://prometheus.io/docs/concepts/metric_types/) should be available at your Prometheus endpoint.
```
Comment (robholland) [inline docker-readme/server.md:377]:
```suggestion
This variable specifies which [type of Prometheus metric](https://prometheus.io/docs/concepts/metric_types/) is used for timer metrics.
```

File: docker-readme/server.md [hunk 2]
```diff
@@ -326,37 +342,63 @@ This variable allows you to pass a frontend security certificate key as an objec
...
 `TEMPORAL_JWT_KEY_REFRESH` (defaults to `1m`)
 
-`TEMPORAL_JWT_PERMISSIONS_CLAIM` (defaults to `permissions`)
+This variable specifies how frequently JWT keys should be refreshed.
 
-`TEMPORAL_AUTH_AUTHORIZER` (defaults to unset)
+`TEMPORAL_JWT_PERMISSIONS_CLAIM` (defaults to `permissions`)
 
-`TEMPORAL_AUTH_CLAIM_MAPPER` (defaults to unset)
+This vairable specifies the name of your JWT permissions claim.
```
Comment (robholland) [inline docker-readme/server.md:401]:
```suggestion
This variable specifies the property name of your JWT permissions claim.
```

**Comment by robholland** [comment]
@axfelix I think this needs your table update?

**Comment by axfelix** [comment]
Done!

**Pull request merged**